packages/api/amplify_api/example/integration_test/main_test.dart

```diff
@@ -29,4 +29,20 @@ void main() async {
     graph_user_pools_test.main(useExistingTestUser: true);
     rest_test.main(useExistingTestUser: true);
   });
+
+  group('amplify_api gen 2', () {
```

Comment:
*Note:* We should discuss when the tests with gen1/cdk are removed. They add some benefit as they test that we are still properly supporting Gen 1 config formats. Integration tests are not an efficient way to test that though. If we feel good about the mapping from Gen 1 -> Gen 2 we could probably remove them.